### PR TITLE
fix: fix user/channel events

### PR DESCRIPTION
**Pull request by peterdeme**

### CODE REVIEW

File: lib/GetStream/StreamChat/Channel.php [hunk 1]
```diff
@@ -432,14 +432,6 @@ public function deleteImage(string $url): StreamResponse
         return $this->client->delete($this->getUrl() . '/image', ["url" => $url]);
     }
 
-    /** @link https://getstream.io/chat/docs/php/custom_events/?language=php
-      * @throws StreamException
-      */
-    public function sendCustomEvent(array $event): StreamResponse
```
Comment (ferhatelmas) [inline lib/GetStream/StreamChat/Channel.php:438]: ok, this is wrong but what are we doing about channel one?

Comment (peterdeme) [reply lib/GetStream/StreamChat/Channel.php:438]: yeah the other (user event) is in `Client.php`

https://github.com/GetStream/stream-chat-php/pull/80/files#diff-7f302ac38049dc9f1a8c5b3a8a13b7d99cb8dd5ae99ceadf8adaf6799323d2f5

Comment (ferhatelmas) [reply lib/GetStream/StreamChat/Channel.php:438]: yes but why are we removing this one?

Comment (peterdeme) [reply lib/GetStream/StreamChat/Channel.php:438]: it was duplicated (my mistake 🤦). there is a other method in Channel.php already.

Comment (ferhatelmas) [reply lib/GetStream/StreamChat/Channel.php:438]: I see, `sendEvent` is there. Good to go then

**Review by ferhatelmas — APPROVED** [review]
LGTM

**Pull request merged**